common/cache.py

```
@@ -0,0 +1,190 @@
+from django.views.decorators.cache import cache_page, never_cache
+from django.views.generic import TemplateView
+from loguru import logger
+from rest_framework import status
+from rest_framework.response import Response
+
+from common.metrics import metrics
+
+
+class CachedTemplateView(TemplateView):
+    @classmethod
+    def as_view(cls, **initkwargs):  # @NoSelf
```

Ruff (0.12.2)

Missing return type annotation for classmethod `as_view`

(ANN206)